build/ballot/1/index.json

```diff
@@ -111,7 +100,7 @@
         {
           "id": 4,
           "name": "Nancy Sidebotham",
-          "photo_url": "https://dl.dropboxusercontent.com/u/36443953/nancy-sidebotham2.JPG",
+          "photo_url": "Provided jpg",
```

Comment:
You'll want to re-import the data since a newer version of the spreadsheet has the URLs that you accidentally removed here